### DISCUSSION TITLE
How to mount prometheus data directory to PVC？

### DISCUSSION THREAD

**Discussion by hillbun** [prometheus-operator/kube-prometheus#970] [Q&A]
How to mount prometheus data directory /prometheus  to PVC？

**Comment by paulfantom** [comment]
We have a configuration snippet for this: https://github.com/prometheus-operator/kube-prometheus/blob/master/examples/prometheus-pvc.jsonnet.

FYI for such multiple questions it is usually better to ask in our slack channel.
